### PR TITLE
run clang-format as part of lint

### PR DESCRIPTION
**Pull request by diracdeltas**
Resolves brave/brave-browser#11912

## Submitter Checklist:

- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed).
- [ ] Requested a security/privacy review as needed.

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: build/commands/scripts/format.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env vpython
+# Copyright (c) 2019 The Brave Authors. All rights reserved.
```
Comment (bbondy) [inline build/commands/scripts/format.py:2]: nit: you can put this to 2020 if you want

**Review by bbondy — APPROVED** [review]
Looks great, thanks!

**Comment by mrobinson** [comment]
I'm having a bit of an issue with this change in https://github.com/brave/brave-core/pull/7154. `git cl format` and `clang-format -i` both modify `chromium_src/chrome/browser/about_flags.cc` in a pretty suspicious way that doesn't pass `npm run lint`. The issue is that lint fails because the file isn't formatted and then after formatting the file, the lint itself (`git cl lint`) fails. The only workaround I see it to use a lot of NOLINT tags in the file, but the formatted version looks truly bizarre.

**Comment by bbondy** [comment]
Could just not run this on that file

**Comment by bbondy** [comment]
I mean run it and then revert the changes to the file.

**Comment by diracdeltas** [comment]
i think you can do `// clang-format off` and `// clang-format on` to make it skip code sections (or in this case, an entire file). in any case i should change this so that lint runs before the format check.

**Comment by iefremov** [comment]
lint checks formatting as well (e.g. lines length, spacing, etc)

**Comment by diracdeltas** [comment]
@iefremov i mean just change the order of commands in lint.py so that it shows the lint errors regardless of whether format check passes

**Comment by mrobinson** [comment]
> i think you can do `// clang-format off` and `// clang-format on` to make it skip code sections (or in this case, an entire file). in any case i should change this so that lint runs before the format check.

Thanks for the suggestion @diracdeltas. I will give this a shot.

**Pull request merged**